Fix:

error: use of undeclared identifier 'errno'

error: use of undeclared identifier 'EOVERFLOW'
--- lib/utime.c.orig	2020-12-31 09:53:25.000000000 -0600
+++ lib/utime.c	2021-01-01 03:12:18.000000000 -0600
@@ -261,6 +261,7 @@
 
 #else
 
+# include <errno.h>
 # include <sys/stat.h>
 # include "filename.h"
 
